e2e/gui/src/test/java/com/epam/pipeline/autotests/LaunchLimitMountsTest.java

```diff
@@ -384,6 +390,46 @@ public void allowToLaunchRunsWithoutMounts() {
                         .close());
     }
 
+    @Test(priority = 4)
+    @TestCase(value = {"TC-PARAMETERS-5"})
+    public void linkStorageWithTheirURLsAndNotOnlyWithTheirIDs() {
+        library()
+                .createStorage(storage5)
+                .selectStorage(storage5)
+                .createFileWithContent("test_file1.txt", "test file1");
+        String storage5ID = Utils.entityIDfromURL();
+        library()
+                .createStorage(storage6)
+                .selectStorage(storage6)
+                .createFileWithContent("test_file1.txt", "test file1");
+        String[] logMessages = {
+                format("Run is launched with mount limits (%s,%s) Only 2 storages will be mounted",
+                        storage5ID, storage6.toLowerCase()),
+                mountStorageMessage(storage5),
+                mountStorageMessage(storage6)
+        };
+
+        library()
+                .createPipeline(pipeline1)
+                .clickOnDraftVersion(pipeline1)
+                .codeTab()
+                .clickOnFile("config.json")
+                .editFile(transferringJsonToObject(profiles -> {
+                    final ConfigurationProfile profile = selectProfileWithName("default", profiles);
+                    profile.configuration.parameters.put("CP_CAP_LIMIT_MOUNTS",
```

Comment:
Let's extract "CP_CAP_LIMIT_MOUNTS" to constant and replace its everywhere

Reply:
Done